src/compaction/leveled.rs

```diff
@@ -20,21 +20,41 @@ fn aggregate_key_range(segments: &[Segment]) -> KeyRange {
 fn pick_minimal_compaction(
     curr_level: &Level,
     next_level: &Level,
-    hidden_set: &HiddenSet,
+    levels: &LevelManifest,
```

Comment:
This can probably stay `&HiddenSet`, and just use `HiddenSet::contains` instead

Reply:
kk